### PR TITLE
Domain model

### PR DESCRIPTION
**Pull request by nk-coding**

### CODE REVIEW

File: scripts/renderDiagrams.js
Comment (delvh) [inline scripts/renderDiagrams.js:1]: Since when do we use pascal case for filenames outside of java?

**Review by delvh — APPROVED** [review]

**Review by lesser1 — APPROVED** [review]

File: scripts/renderDiagrams.js
Comment (nk-coding) [reply scripts/renderDiagrams.js:1]: for js/ts, I've commonly seen
- `renderDiagrams.js`
- `render-diagrams.js`
- `render_diagrams.js`

I don't really care which one we use (although I don't like the underscore one). Pascal case is just what I used historically the most. But I think we could get a team opinion in this, in particular for the team members using js/ts in the future.

Comment (lesser1) [reply scripts/renderDiagrams.js:1]: I don't care as long as we stick with one convention

File: diagrams/domainModelBC.hyl
Comment (lesser1) [inline diagrams/domainModelBC.hyl:1]: Regarding the relationship between Address and Shipment, I would argue that the direction Address has zero to many Shipments is not needed and should be removed in order to keep the model as simple as possible. The same logic applies to "a ShipmentMethod has zero to many Shipments". But I do not insist on this change...

Comment (nk-coding) [reply diagrams/domainModelBC.hyl:1]: done

**Pull request merged**